kloppy/tests/files/tracab_raw.json

```diff
@@ -0,0 +1,4406 @@
+        {
+          "Team": -1,
+          "JerseyNumber": 0,
+          "X": -4069,
+          "Y": -2965,
+          "Speed": 0
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 7,
+          "X": -3310,
+          "Y": 1325,
+          "Speed": 1.75
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 8,
+          "X": -2202,
+          "Y": 834,
+          "Speed": 1.9
+        },
+        {
+          "Team": 0,
+          "JerseyNumber": 12,
+          "X": -5125,
+          "Y": 407,
+          "Speed": 0.82
+        },
+        {
+          "Team": -1,
+          "JerseyNumber": 0,
+          "X": 2545,
+          "Y": -3186,
+          "Speed": 0
+        },
+        {
+          "Team": 0,
+          "JerseyNumber": 4,
+          "X": -4004,
+          "Y": 18,
+          "Speed": 1.23
+        },
+        {
+          "Team": 0,
+          "JerseyNumber": 27,
+          "X": -3623,
+          "Y": -712,
+          "Speed": 1.66
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 16,
+          "X": -3140,
+          "Y": -647,
+          "Speed": 1.29
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 55,
+          "X": -2140,
+          "Y": -1385,
+          "Speed": 1.49
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 20,
+          "X": -2804,
+          "Y": -298,
+          "Speed": 1.34
+        },
+        {
+          "Team": 0,
+          "JerseyNumber": 26,
+          "X": -3163,
+          "Y": 604,
+          "Speed": 1.31
+        },
+        {
+          "Team": 4,
+          "JerseyNumber": 0,
+          "X": 5550,
+          "Y": 4400,
+          "Speed": 0
+        },
+        {
+          "Team": 1,
+          "JerseyNumber": 77,
+          "X": -3308,
+          "Y": -349,
+          "Speed": 1.74
+        }
+      ],
+      "BallPosition": [
+        {
+          "BallContactInfo": "",
+          "BallOwningTeam": "H",
+          "BallStatus": "Dead",
+          "Speed": 121.28,
+          "X": -4486,
+          "Y": 425,
+          "Z": 4
+        },
+        {
+          "BallContactInfo": "",
+          "BallOwningTeam": "H",
+          "BallStatus": "Dead",
+          "Speed": 121.28,
+          "X": 5550,
+          "Y": 4400,
+          "Z": 0
+        },
+        {
+          "BallContactInfo": "",
+          "BallOwningTeam": "H",
+          "BallStatus": "Dead",
+          "Speed": 121.28,
+          "X": 5550,
+          "Y": 4400,
+          "Z": 0
+        },
+        {
+          "BallContactInfo": "",
+          "BallOwningTeam": "H",
+          "BallStatus": "Dead",
+          "Speed": 121.28,
+          "X": 5550,
+          "Y": 4400,
+          "Z": 0
+        },
+        {
+          "BallContactInfo": "",
+          "BallOwningTeam": "H",
+          "BallStatus": "Dead",
+          "Speed": 121.28,
+          "X": 5550,
+          "Y": 4400,
+          "Z": 0
+        }
+      ]
```

Comment:
I don't really understand why we in the raw data, we have per frame 5 ball positions. It seems that the first ball position contains the right information, and the other 4 are duplicates but with the max X and Y values? Not sure what is going on.

Reply:
Do your own data files have the same structure as this example data file?

Reply:
Yes

Reply:
Great. Thanks for checking with them!